Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook

```diff
@@ -3516,7 +3474,2305 @@
+                      "xAxis": "timestamp_t",
+                      "yAxis": [
+                        "FREE_MEM_MB_d",
+                        "ACT_WPS_d"
+                      ],
+                      "seriesLabelSettings": [
+                        {
+                          "seriesName": "ACT_DIA_d",
+                          "label": "Active Dialog  WPs"
+                        },
+                        {
+                          "seriesName": "ACT_WPS_d",
+                          "label": "Active WPs​"
+                        },
+                        {
+                          "seriesName": "TRFC_FREE_d",
+                          "label": "Available RFC WPs"
+                        },
+                        {
+                          "seriesName": "FREE_MEM_MB_d",
+                          "label": "Free WPs"
+                        },
+                        {
+                          "seriesName": "FREE_MEM_MB_INC_FS_d",
+                          "label": "Free Memory including File System Cache (MB)"
+                        }
+                      ]
+                    }
+                  },
+                  "name": "ActiveWorkprocess (SM66)",
+                  "styleSettings": {
+                    "showBorder": true
+                  }
+                },
+                {
+                  "type": 12,
+                  "content": {
+                    "version": "NotebookGroup/1.0",
+                    "groupType": "editable",
+                    "items": [
+                      {
+                        "type": 3,
+                        "content": {
+                          "version": "KqlItem/1.0",
+                          "query": "let appServerList = dynamic([{CpuMemApplicationServer}]);\r\nSapNetweaver_SMON_CL\r\n| where timestamp_t between (todatetime({CpuMemTimeRange:start}) .. todatetime({CpuMemTimeRange:end}))\r\n| where SID_s == '{CpuMemSID}'\r\n| project USERS_d, SESSIONS_d, SID_s, hostname_s, instanceNr_s, timestamp_t,appServerList\r\n| extend instanceNr_s = iff(strlen(instanceNr_s) == 1, strcat(\"0\", instanceNr_s), instanceNr_s)\r\n| extend appServer = strcat(hostname_s, \"_\", SID_s, \"_\", instanceNr_s)\r\n| where array_length(appServerList) == 0 or strcat(hostname_s, \"_\", instanceNr_s) in (appServerList)\r\n| project USERS_d, SESSIONS_d, timestamp_t\r\n",
+                          "size": 0,
+                          "title": "User Logons and Sessions",
+                          "queryType": 0,
+                          "resourceType": "microsoft.operationalinsights/workspaces",
+                          "visualization": "linechart",
+                          "chartSettings": {
+                            "xAxis": "timestamp_t",
+                            "yAxis": [
+                              "SESSIONS_d",
+                              "USERS_d"
+                            ],
+                            "seriesLabelSettings": [
```

Comment:
a think a lot of copy and paste artifacts here in series labels, lets remove all of them except for the USERS_d and SESSIONS_d